### ISSUE TITLE
Clarification on cost volume preparation

### ISSUE THREAD

**Issue by InnovArul** [Jiankai-Sun/GC-Net#2]
Hi, I am trying to understand the cost volume calculation.
I have a doubt in the code. especially in https://github.com/Jiankai-Sun/GC-Net/blob/master/network.py#L193

Every pixel _(i, j)_ {i = rows, j = columns} in Left image shall be compared with the pixels _(i, j-k)_ where k = [0 to disparity] of the right image.

Hence, in my understanding, the particular line to slice the right feature map shall be:

right_feature = tf.slice(right_feature, [0, **0**, 0], [HEIGHT/2, WIDTH/2-d, 1])

What is your opinion on this?
Thanks in advance!

**Comment by stalin18** [comment]
@InnovArul I agree with you on this point. Even I noticed this (I think this is a mistake).
